pom.xml

```diff
@@ -44,9 +44,258 @@
         <jenkins.version>2.361.4</jenkins.version>
         <groovy.version>2.4.21</groovy.version> <!-- TODO: Add org.codehaus.groovy:groovy and org.codehaus.groovy:groovy:sources to Jenkins core BOM so this can be deleted? (currently it only specifies groovy-all) -->
     </properties>
+<<<<<<< HEAD
```

Comment:
@MartinKosicky Sorry, but #612 turned this into a multi-module project, so you've got an unresolved merge conflict here now. I think you want to delete all of the changes in this file and apply any dependency updates to `plugin/pom.xml` instead.

Reply:
hmm im getting "child.scm.connection.inherit.append.path is not allowed here in pom.xml"

Reply:
My guess is either that you need to update Maven or there is some bug with the way the IDEA tries to interpret `pom.xml` on its own now that this is a more complex project. I think there is a setting somewhere where you can tell IDEA to use Maven for the build and bypass the default behavior, but I am not 100% sure.